content/includes/veteran-navigation.html

```diff
@@ -267,6 +267,7 @@
               <!--<li><a class="login-required" href="/veteran-id-card/how-to-get" onClick="reportHeaderNav('Request Your Records and ID');">Apply for a Veteran ID Card</a></li>-->
               <li><a href="/discharge-upgrade-instructions/" onClick="reportHeaderNav('Request Your Records and ID->Apply For Discharge Upgrade');">How to Apply for a Discharge Upgrade</a></li>
               <li><a href="/health-care/health-records/" onClick="reportHeaderNav('Request Your Records and ID->Health Care');">Get Your VA Health Records</a></li>
+              <li><a href="/download-va-letters/" onClick="reportHeaderNav('Request Your Records and ID->Health Care');">Download VA letters</a></li>
```

Comment:
It looks like we need to update this content as well, not sure what we should replace it with, perhaps "Download Your VA Letters"?

Reply:
Perhaps  “Request Your Records and ID->Download Your VA Letters” is more consistent?